app/models/concerns/taxonomix.rb

```diff
@@ -195,9 +195,7 @@ def children_of_selected_organization_ids
   # for existing records it would block saving global objects if taxonomies were not changed
   # for new records this would allow to create global objects for non global users
   def ensure_taxonomies_not_escalated
-    taxonomies = []
-    taxonomies << Organization if SETTINGS[:organizations_enabled]
-    taxonomies << Location if SETTINGS[:locations_enabled]
+    taxonomies = [Organization, Location]
```

Comment:
nitpick: could you move it to a helper? there are few identical usages.

Reply:
Would this be easier to understand?

```ruby
    taxonomies = Taxonomy.subclasses
```

Reply:
I'd prefer a dedicated method. Don't we have `Taxonomy.all_enabled_taxonomies` or something? I get, that the enabled doesn't make sense anymore. But something similar would be good. `Taxonomy.all_kinds` or `Taxonomy.all_available` or ...

Reply:
Ah, should have read the whole PR. I do like `enabled_taxonomies`, just the naming needs an update imho.

Reply:
How about `Taxonomy.types`?

Reply:
Sure, why not.

Reply:
added and changed other places that had the same array to use it.